src/main/java/com/xloop/resourceloop/authenticationservice/Controller/AuthController.java

```diff
@@ -0,0 +1,35 @@
+package com.xloop.resourceloop.authenticationservice.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.xloop.resourceloop.authenticationservice.JPARepository.UserRepository;
+import com.xloop.resourceloop.authenticationservice.Model.User;
+
+@RestController
+@RequestMapping("/auth")
+public class AuthController {
+    @Autowired
+    private UserRepository userRepo;
+
+    @PostMapping("/register")
+    public ResponseEntity<String> register(@RequestBody User user){
+        User exist_user = userRepo.findByEmail(user.getEmail());
```

Comment:
@hunainparekh2211006KHICND try to be consistent with the casing - 
`exist_user` is snake case, but `camelCase` is used elsewhere and is the  standard in Java e.g `existUser` or better `userExists`